docs/TextColumns.md

```diff
@@ -0,0 +1,77 @@
+_New in [:octicons-tag-24: 2.7.7](https://github.com/py-pdf/fpdf2/blob/master/CHANGELOG.md)_
+
+**Notice:** As of fpdf2 release 2.7.7, this is an experimental feature. Both the API and the functionality may change before it is finalized, without prior notice.
+
+
+## Text Columns ##
+
+The `FPDF.text_columns()` method allows to create columnar layouts, with one or several columns. Columns will always be of equal width.
+
+Beyond the parameters common to all text regions, the following are available for text columns:
+
+* l_margin (float, optional) - override the current left page margin.
+* r_margin (float, optional) - override the current right page margin.
+
+Only for `FPDF.text_columns()`:
+
+* ncols (float, optional) - the number of columns to generate (Default: 2).
+* gutter (float, optional) - the space required between each two columns (Default 10).
```

Comment:
This spacing is applied both horizontally & vertically?
Maybe the documentation should clearly state it?

Reply:
you seem to be thinking about a table... Columns only have horizontal spacing.

Reply:
Oh alright!
It might be worth being explicit there, for dummies like me 😅